website/allowed-subcategories.txt

```diff
@@ -50,6 +50,7 @@ Elastic Load Balancing v2 (ALB/NLB)
 Elastic Map Reduce (EMR)
 Elastic Transcoder
 ElasticSearch
+EventBridge (fka CloudWatch Events)
```

Comment:
I'm not sure if `fka` (formerly known as) is a globally known acronym, especially for non-native English speakers. I would recommend just dropping it.